README.md

```diff
@@ -1,24 +1,43 @@
 # OVAL visualization as graph
+Understanding result in the blink of an eye
+
 [![Scrutinizer Code Quality](https://scrutinizer-ci.com/g/OpenSCAP/OVAL-visualization-as-graph/badges/quality-score.png?b=master)](https://scrutinizer-ci.com/g/OpenSCAP/OVAL-visualization-as-graph/?branch=master) [![Code Coverage](https://scrutinizer-ci.com/g/OpenSCAP/OVAL-visualization-as-graph/badges/coverage.png?b=master)](https://scrutinizer-ci.com/g/OpenSCAP/OVAL-visualization-as-graph/?branch=master) [![Build Status](https://scrutinizer-ci.com/g/OpenSCAP/OVAL-visualization-as-graph/badges/build.png?b=master)](https://scrutinizer-ci.com/g/OpenSCAP/OVAL-visualization-as-graph/build-status/master) [![Code Intelligence Status](https://scrutinizer-ci.com/g/OpenSCAP/OVAL-visualization-as-graph/badges/code-intelligence.svg?b=master)](https://scrutinizer-ci.com/code-intelligence)
 
-Understanding result in the blink of an eye
+## Visualization of SCAP rule evaluation results
+This client generate tree graph from ARF xml report from OpenSCAP scan.
+## Prerequisites
+##### Minimal requirements
+   - **python3**
+   - lxml
+   
+##### Recommended requirements
+  * include minimal requirements
+  * inquirer
+
+### Installation
+
+```bash
+git clone https://github.com/OpenSCAP/OVAL-visualization-as-graph.git
+cd OVAL-visualization-as-graph
+
+# Install package with nice cli futures (recommended)
+python3 setup.py install ".[niceCli]"
```

Comment:
Why is there the dot after install? And is the command really correct? I get an `invalid command name '.[niceCli]'` error